matchzoo/models/cdssm_model.py

```diff
@@ -0,0 +1,95 @@
+"""An implementation of CDSSM (CLSM) model."""
+
+from matchzoo import engine
+
+from keras import Model
+from keras.layers import Input, Conv1D, MaxPooling1D, Dot, Dense, Flatten
+
+
+class CDSSMModel(engine.BaseModel):
+    """
+    Convolutional deep structured semantic model.
+
+    Examples:
+        >>> model = CDSSMModel()
+        >>> model.guess_and_fill_missing_params()
+        >>> model.build()
+
+    """
+
+    @classmethod
+    def get_default_params(cls) -> engine.ParamTable:
+        """:return: model default parameters."""
+        params = super().get_default_params()
+        params['optimizer'] = 'sgd'
```

Comment:
the default 'sgd' optimizer could not yield a good result if we do not adjust learning rate. 
'sgd' -> 'adam' is better for default parameter.

Reply:
let's keep the default parameters the same as original paper, and leave the hyper-parameters tunning to the user. @pl8787

Reply:
I think the default parameters is better. The better settings of parameter can be shown in the documentation of each model.